README.md

```diff
@@ -1 +1,194 @@
-# Awesome-Books-Project
\ No newline at end of file
+<a name="readme-top"></a>
+
+<!--
+HOW TO USE:
+This is an example of how you may give instructions on setting up your project locally.
+
+Modify this file to match your project and remove sections that don't apply.
+
+REQUIRED SECTIONS:
+- Table of Contents
+- About the Project
+  - Built With
+  - Live Demo
+- Getting Started
+- Authors
+- Future Features
+- Contributing
+- Show your support
+- Acknowledgements
+- License
+
+After you're finished please remove all the comments and instructions!
+-->
+
+<div align="center">
+
+  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
+  <h3><b>Microverse README Template</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Project Name
+
+#### Awesome Books Project
+
+## 🛠 Built With <a name="built-with"></a>
+
+- HTML, CSS, and JavaScript
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+> Describe between 1-3 key features of the application.
+
+- The project has been made using personalized ideas but following [design guidelines](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) for colors, typography, and layout.
+- The project is done in plane CSS. No CSS framework has been used for styling.
+- Only **Home** and **About** page have been designed as per the requirement of the project.
+- Each of the pages have been adopted for 2 different screen sizes - **Mobile** *768px wide* and **Desktop** *768px or wider* as per the requirement.
+- Interactions (e.g., *Links* and *Mobile menu*) are put.
+- Dynamic page (*Featured Clubs*) is created using JavaScript.
+- Readme file has been adopted as per the requirements.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
```

Comment:
- [ ] Hello team good job so far 👏. However, your readme file does not reflect your project 😄. Please would you like to document it based on the project? Actually, your readme file should only include only the relevant pieces of information that apply to your project. 👌